.github/workflows/client-type-check.yaml

```diff
@@ -33,9 +33,8 @@ jobs:
         run: |
           cd client
           set +e
-          godot --project . --headless --quit-after 100 -e 2>/dev/null
-          godot --project . --headless --quit-after 100 -e 2>/dev/null
-          godot --project . --headless --quit-after 100 -e 2>/dev/null
+          godot --project . --headless --import -e 2>/dev/null
+          godot --project . --headless --import -e 2>/dev/null
```

Comment:
it improved, but still we need to run it twice, I think the problem is that error we get on the first run allways. :(

Reply:
Could you create an issue for that problem and reference this workaround (so it doesn't get forgotten)?